woocommerce/woocommerce
# Add support for Twenty Twenty-One theme

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds support for the Twenty Twenty-One theme.

Closes #28134 

### How to test the changes in this Pull Request:

1. Install the Twenty Twenty-One theme by following the [instructions on the Github Repo](https://github.com/WordPress/twentytwentyone#installation).
2. Move the theme from `twentytwentyone-trunk` to `twentytwentyone` and make sure it is still active or activate it after doing this. This is necessary because of the checks in `theme_support_includes`.
3. Apply this PR 

### Other information:




### Changelog entry

> Enhancement: Add compatibility for Twenty Twenty-One theme.


Issues fixed, with screenshots:

Fixes #28276 
![image](https://user-images.githubusercontent.com/363749/99274031-692a9f00-27ef-11eb-8ce6-d3c01b4d43f9.png)

Fixes #28274 
![image](https://user-images.githubusercontent.com/363749/99274241-b3138500-27ef-11eb-9b38-bd76a40d530d.png)

Fixes #28273 
![image](https://user-images.githubusercontent.com/363749/99274371-e0603300-27ef-11eb-999a-56540f6838bc.png)

Fixes #28272
![image](https://user-images.githubusercontent.com/363749/99274498-05ed3c80-27f0-11eb-8b55-39990d211b02.png)
